client/src/components/modal-forms/EditSiteForm.js

```diff
@@ -51,6 +52,7 @@ export const EditSiteForm = ({
             />
             <Field name='address' component={RenderTextField} label='* Street Address' />
             <Field name='city' component={RenderTextField} label='* City' />
+            <Field name='postalCode' component={RenderTextField} label='* Postal Code' />
```

Comment:
How come the reordering? I would put site type under postal code, I don't think it looks great under the radio

Reply:
I did this as it seemed like postalCode should go with address/city but was being broken up by the is HA radio buttons. 

Show i move postal code back below isHA? or just move siteType above isHA and keep postalCode under city?